controllers/cache/constants.go

```diff
@@ -0,0 +1,20 @@
+package cache
+
+const (
+	GetRedisClientError = "Get redis client error"
+	CheckRedisHealthError = "Check redis health error"
+	CreateComponentSecretError = "Create component secret error"
+
+	GenerateRedisCrError = "Generate redis cr error"
```

Comment:
```suggestion
	GenerateRedisCrError = "Generate redis CR error"
```

I think all `cr` should be changed to `CR`

Reply:
@Ghostbaby 

cr->CR?